websites/M/MangaWorld/presence.ts

```diff
@@ -1,10 +1,10 @@
 const presence = new Presence({
 		clientId: "860298084512038964"
 	}),
-	browsingTimestamp = Math.floor(Date.now() / 1000);
+	browsingStamp = Math.floor(Date.now() / 1000);
 
 presence.on("UpdateData", async () => {
-	const presenceData: PresenceData = {
+	const data: PresenceData = {
```

Comment:
`presenceData` is preferred

Reply:
Fixed in commit d0770c80e834f054598a5613c64867d2763c2987